diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index 759f6c8..f57c9c9 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -153,6 +153,9 @@ $(foreach p, $(call to-upper, $(SSI_PARTITIONS)), \
     $(eval BOARD_$(p)IMAGE_PARTITION_RESERVED_SIZE := $(SSI_PARTITIONS_RESERVED_SIZE)))
 $(foreach p, $(call to-upper, $(TREBLE_PARTITIONS)), \
     $(eval BOARD_$(p)IMAGE_PARTITION_RESERVED_SIZE := 30720000))
+BOARD_VENDORIMAGE_PARTITION_RESERVED_SIZE := 1224000000
+BOARD_PRODUCTIMAGE_PARTITION_RESERVED_SIZE := 307200000
+BOARD_VENDORIMAGE_EXTFS_INODE_COUNT := -1
 
 # Platform
 BOARD_VENDOR := xiaomi
